azavea/raster-foundry
# Feature/filter bad landsat images

## Overview

This is a collection of small changes:

- Making links to db and redis `external_links` in the airflow docker-compose file was necessary because otherwise those containers couldn't build
- Setting `minCloudCover` to 0 if it wasn't present was necessary to filter out bad landsat scenes and was issue number #834
- Reverting toggle selection on scene click in color correct mode was a design note and issue #847 
- Zooming in and out in mosaic mode when drawing and not drawing masks was issue #853 

They're in separate commits for easy viewing.

### Checklist


## Testing Instructions

- `./scripts/server --with-airflow` shouldn't complain about having a link to undefined services
- you shouldn't see any missingno landsat 8 scenes no matter how far you browse
- clicking on a scene in color correct view should take you into the color correct panel
- clicking on a scene in mosaic mode should zoom to that scene, and hitting the x on drawing a mask on that scene should zoom out to the project again

Closes #834
Closes #847
Closes #853
